RSS Reader/RSS Reader/Default/AppDelegate.m

```diff
@@ -0,0 +1,69 @@
+//
+//  AppDelegate.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/17/20.
+//
+
+#import "AppDelegate.h"
+#import "RSSRTopicsListTableViewController.h"
+#import "RSSRNetworkService.h"
+#import "RSSRXMLParser.h"
+#import "RSSRTopicsListPresenter.h"
+
+@interface AppDelegate ()
+
+@end
+
+@implementation AppDelegate
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    if (@available(iOS 13.0, *)) {
+        
+    } else {
+        UIWindow *window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
+        self.window = window;
+        self.window.backgroundColor = UIColor.whiteColor;
```

Comment:
This is wrong syntax for Objective-C. In case you are trying to access property, please, use dot syntax. In case you are calling a function address it using whitespace and square brackets.